lib/vmpooler/generic_connection_pool.rb

```diff
@@ -15,35 +15,17 @@ def initialize(options = {}, &block)
         @metric_prefix = 'connectionpool' if @metric_prefix.nil? || @metric_prefix == ''
       end
 
-      if Thread.respond_to?(:handle_interrupt)
-        # MRI
```

Comment:
Heh, I remember we discussed this code being fishy when debugging the Jruby issue. Isn't it essentially unreachable?

Reply:
Yes. With all of our tested MRI and jruby versions they respond to `:handle_interrupt` so I think it is past its time. We used to run old jruby with vmpooler, which is probably why this is here.

Reply:
💀 RIP dead code 💀